### PR TITLE
feat: handle nsfw flag & added monospace font

### PR DESCRIPTION
**Pull request by Mar0xy**
Add NSFW Switch to channel settings overview.
requires following PRs:
[Revolt.js#15](https://github.com/revoltchat/revolt.js/pull/15)
[Delta#90](https://github.com/revoltchat/delta/pull/90)
This also adds JetBrains Mono ([License](https://github.com/JetBrains/JetBrainsMono/blob/master/OFL.txt)) which was requested by users

Preview:

https://user-images.githubusercontent.com/8841466/133856712-83d1de2a-e895-4473-aa09-4f2759e52dee.mp4

### CODE REVIEW

File: src/context/Theme.tsx [hunk 1]
```diff
@@ -62,7 +62,8 @@ export type MonospaceFonts =
     | "Roboto Mono"
     | "Source Code Pro"
     | "Space Mono"
-    | "Ubuntu Mono";
+    | "Ubuntu Mono"
+    | "Jetbrains Mono";
```
Comment (Rexogamer) [inline src/context/Theme.tsx:66]: `Jetbrains` should be written as `JetBrains`

File: src/context/Theme.tsx [hunk 2]
```diff
@@ -211,6 +212,10 @@ export const MONOSPACE_FONTS: Record<
         name: "Ubuntu Mono",
         load: () => import("@fontsource/ubuntu-mono/400.css"),
     },
+    "Jetbrains Mono": {
+        name: "Jetbrains Mono",
```
Comment (Rexogamer) [inline src/context/Theme.tsx:216]: as above

**Review by Rexogamer — COMMENTED** [review]
Minor grammar nitpick:

**Comment by Mar0xy** [comment]
Those were really minor nitpicks but I don't mind it as JetBrains is written like that my brain was just half asleep at the time of adding in the monofont so I ended up forgetting to capitalize the B

**Review by insertish — APPROVED** [review]
Note: Need to add i18n on merge.

**Pull request merged**